docs/contributing/contributing-code/contributing-code-control-plane/running-controlplane-locally.md

```diff
@@ -193,7 +205,6 @@ You should be able to successfully the following commands from the Radius reposi
 
 ```sh
 ls ../deployment-engine/src
-ls ../deployment-engine/submodules/bicep-extensibility/src
```

Comment:
Why is this not needed anymore?

Reply:
Because we removed submodules from DE. It can be linked to the other comment you had above.